platforms/anchor-platform/admin-guide/sep24/configuration.mdx

```diff
@@ -109,6 +113,8 @@ SECRET_SEP24_MORE_INFO_URL_JWT_SECRET="your encryption key shared with your busi
 
 `SECRET_SEP24_INTERACTIVE_URL_JWT_SECRET` and `SECRET_SEP24_MORE_INFO_URL_JWT_SECRET` are encryption keys that the Anchor Platform will use to generate short-lived tokens it will add to the URLs provided to the wallet. Your business server must also have these keys in its environment so it can verify the token's signature.
 
+`SEP24_INITIAL_USER_DEADLINE_SECONDS` is the time in seconds a user has to act before the transaction automatically moves to the next status. It determines the `user_action_required_by` field, which indicates the deadline for statuses with automatic movement. Check [JSON-RPC Methods][json-rpc-methods] for usage examples.
+
```

Comment:
I must have missed the PR implementing `user_action_required_by` so I'm missing context, but why do we have a configurable default value for this? Wouldn't the time the user has to act be highly dependent on the specific situation? I would've thought we would always require the business to set the time in each RPC request.

Also when you say "automatically moves to the next status", the business still has to make an RPC request to move the transaction to the next status right? The anchor platform doesn't monitor the time and move the transaction status on its own.

Reply:
Do you min adding an example of GET /transactions usage?